src/chocolatey.resources/helpers/functions/Get-ChocolateyWebFile.ps1

```diff
@@ -261,8 +261,18 @@ param(
 
   # If we're on 32 bit or attempting to force 32 bit and there is no
   # 32 bit url, we need to throw an error.
-  if ($url -eq $null -or $url -eq '') {
-    throw "This package does not support $bitPackage architecture."

```

Comment:
This error message is still relevant in some situations. Specifically, it still should be shown "If we're on 32 bit or attempting to force 32 bit and there is no 32 bit url", but there is a 64 bit URL provided.

Reply:
Sure, I restore this message but changed conditions :)